docs/source/docker/getting_started/hardware_recommendations.rst

```diff
@@ -0,0 +1,82 @@
+.. _ref-hardware-recommendations:
+
+Hardware recommendations
+========================
+
+Usually, the Lightly worker is run on a cloud machine,
+which is spin up specifically to run the docker.
+Our recommendations on the hardware configuration of this machine are
+based on three criteria:
+
+- speed: The worker should process your dataset as quickly a possible.
+- cost-effectiveness: The machine should be cheap to rent.
+- stability: The worker should not crash because it runs out of memory.
+
+As a starting point, we recommend a machine with the following configuration:
+
+- 8 vCPUs
+- 30-32 GB of system memory
+- a NVIDIA-GPU with 16GB of video memory, e.g. the T4 GPU.
```

Comment:
Memory is not important. I'd go for "at least 8 GB of video memory as this covers most machines. after e.g. I'd mention also `P100, P4, V100, A10` or newer